packages/types/src/scope.ts

```diff
@@ -13,8 +13,8 @@ export type CaptureContext = Scope | Partial<ScopeContext> | ((scope: Scope) =>
 export interface ScopeContext {
   user: User;
   level: Severity;
-  extra: { [key: string]: any };
-  contexts: { [key: string]: any };
```

Comment:
This should match the definition in the `Event` type https://github.com/getsentry/sentry-javascript/blob/a575c83942722c7ac7a4c2443b7a01184e74b36a/packages/types/src/event.ts#L34-L36

How about we create a specific type and use it in Event and ScopeContext?